Experimental/DistributedAuthoritySample/ProjectSettings/ProjectSettings.asset

```diff
@@ -176,7 +176,7 @@ PlayerSettings:
   AndroidMinSdkVersion: 23
```

Comment:
What I changed here is that I set the Active Input Handling in Player settings to "Both", so that both your player movement scripts and Dave and my temporary one works while we still have it. Later on we can reset it to only the newer Input system.

Reply:
It seems like there's a difference in our line endings settings, so the whole file was marked as changed.
Let's resolve that item offline, but if you can add just that one line change instead that'd be great.